docs/plot.rst

```diff
@@ -0,0 +1,111 @@
+
+   mdbenchmark plot --output-name my_benchmark_plot
+
+You can also specify a filetype with the ``--output-format`` option.
+Supports file formats are ``png``, ``pdf``, ``svg`` and ``ps``. To set the file format to PNG, run::
+
+   mdbenchmark plot --output-format png
+
+Filter what to plot
+-------------------
+
+To filter your data from the given CSV file(s) you can use the following commands. These can be mixed together as you like.
+
+Exclude CPU or GPU benchmarks from plots
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default both GPU and CPU data will be plotted. To create a plot without the GPU benchmarks run::
+
+   mdbenchmark plot --no-gpu
+
+To create a plot without CPU benchmarks run::
+
+   mdbenchmark plot --no-cpu
+
+Filter by MD engine
+~~~~~~~~~~~~~~~~~~~
+
+If you have run benchmarks for different MD engines, but want to only plot a subset of these, you can do so with the ``--module`` option. For example, if you wanted to plot all benchmarks for the two modules ``gromacs/2018.2`` and ``gromacs/2016.4``, you would run:: 
+
+   mdbenchmark plot --module gromacs/2018.2 --module gromacs/2016.4
+
+Filter by job template
+~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to filter your benchmarks by the job template that was used for any given benchmark with the ``--template`` option. To only plot benchmarks that were run with the ``draco`` job template, run::
+
+   mdbenchmark plot --template draco
+
+.. note::
+   Both ``--template`` and ``--host`` may be used interchangeably. While some users might think of job templates as one specific template for their  cluster, others might think of them as a bundle of templates with different settings for the same cluster. Either view is correct, and thus we provide both options, but prefer ``--template``.
+
+Changing axis labels from nodes to cores
+----------------------------------------
+
+To change the x-axis label from number of nodes to number of cores you can run::
+
+   mdbenchmark plot --plot-cores
+
+Hiding the linear fit
+---------------------
+
+To create a plot without any linear fit, use the ``--no-fit`` option::
+
+  mdbenchmark plot --no-fit
+
+Changing font size
+------------------
+
+Font size can be adjusted with the ``--font-size`` option. The default is ``16pt``::
+
+  mdbenchmark plot --font-size
+
+Adjusting plot resolution (DPI)
+-------------------------------
+
+The plot resolution can be changed with the ``--dpi`` option. The default ist ``300``::
+
+  mdbenchmark plot --dpi
+
+Customizing ticks on the x-axis
+-------------------------------
+
+It is possible to change the frequence of ticks on the x-axis. To do this, call the ``--xtick-step`` option::
+
+  mdbenchmark plot --xtick-step
+
+Removing the watemark
+---------------------
+
+By default a small watermark will be placed in the top left corner of every plot. To disable this, use the ``--no-watermark`` option::
+
+  mdbenchmark plot --no-watermark
```

Comment:
we should add a note that it would be nice if users left this or cited the zenodo DOI to give us credit when they use the tool.